docs/install/install-from-packages.md

```diff
@@ -0,0 +1,279 @@
+---
+myst:
+  html_meta:
+    "description": "How to install Plone 6 from its packages."
+    "property=og:description": "How to install Plone 6 from its packages."
+    "property=og:title": "Install Plone from its packages"
+    "keywords": "Plone, Plone 6, install, pip, packages, source, cookiecutter"
+---
+
+
+(install-packages-1-label)=
+
+# Install Plone from its packages
+ 
+When you want full control over development or deployment, installing Plone from its packages is a good option.
+
+
+(install-packages-system-requirements-label)=
+
+## System requirements
+
+The hardware requirements below give a rough estimate of the minimum hardware setup needed for a Plone server.
+
+Add-on products and caching solutions may increase RAM requirements.
+
+A single Plone installation is able to run many Plone sites.
+You may host multiple Plone sites on the same server.
+
+-   Almost any modern operating system, including Linux, macOS, and Windows, but a UNIX-based operating system is recommended.
+-   Minimum 256 MB RAM and 512 MB of swap space per Plone site is required.
+    2 GB or more RAM per Plone site is recommended.
+-   Minimum 512 MB hard disk space is required.
+    40 GB or more hard disk space is recommended.
+
+
+(install-packages-prerequisites-label)=
+
+### Pre-requisites for installation
+
+-   Python 3.8, 3.9, or 3.10.
+-   Cookiecutter
+-   Node.JS
+-   nvm
+-   Yarn
+-   ```{todo}
+    List any system libraries, such as `make`, `Xcode`, and so on.
+    ```
+
+Installing Python is beyond the scope of this documentation.
+However, it is recommended to use a Python version manager, [`pyenv`](https://github.com/pyenv/pyenv) that allows you to install multiple versions of Python on your development environment without destroying your system's Python.
+
+Install or upgrade {term}`Cookiecutter` and {term}`mxdev` in your user's Python:
+
+```shell
+pip install --user --upgrade cookiecutter
+```
+
+{ref}`Install nvm and Node.js documentation <frontend-getting-started-install-nvm-label>`.
+
+{ref}`Install Yarn documentation <frontend-getting-started-yarn-label>`.
+
+
+(install-packages-install-label)=
+
+## Install Plone 6
+
+We install Plone 6 with {term}`pip`, {term}`Cookiecutter`, {term}`mxdev`, {term}`make`, and other developer tools.
+
+```{note}
+We do not maintain documentation for installing Plone 6 or later with `buildout`.
+For Plone 5, `buildout` was the preferred installation method.
+You can read the [documentation of how to install Plone 5 with `buildout`](https://docs.plone.org/manage/installing/installation_minimal_buildout.html), and adapt it to your needs for Plone 6.
+```
+
+Create a new directory to hold your project, and make it your current directory.
+
+```shell
+mkdir my_project
+cd my_project
+```
+
+Run `cookiecutter` to create a Plone project skeleton using the cookiecutter {term}`cookiecutter-plone-starter` with the following command.
```

Comment:
If one is not familiar with cookicutter the following sentence is over-cookiecuttered:
```suggestion
Run `cookiecutter` to create a Plone project skeleton using the template {term}`cookiecutter-plone-starter` with the following command.
```